pkg/controller/storagecluster/storageclasses.go

```diff
@@ -66,8 +66,6 @@ func (r *ReconcileStorageCluster) createStorageClasses(scs []*storagev1.StorageC
 					return err
 				}
 				reqLogger.Info(fmt.Sprintf("Creating StorageClass %s", sc.Name))
-				existing.ObjectMeta.OwnerReferences = sc.ObjectMeta.OwnerReferences
-				sc.ObjectMeta = existing.ObjectMeta
```

Comment:
Don't we want to keep anything from the existing storage class like labels and annotations etc.

Reply:
No. We are reconciling, we own the StorageClass, that means that any modifications to the StorageClass must be overridden.